### PR TITLE
[TACHYON-989] Update the run tests script.

### PR DESCRIPTION
**Pull request by calvinjia**
Follow up to #1636 to make `runTests` work again.

### CODE REVIEW

File: examples/src/main/java/tachyon/examples/BasicRawTableOperations.java [hunk 1]
```diff
@@ -127,11 +128,12 @@ private void write(TachyonFS tachyonClient) throws IOException {
 
   public static void main(String[] args) throws IllegalArgumentException {
     if (args.length != 3) {
-      System.out.println("java -cp " + Constants.TACHYON_JAR
-          + " tachyon.examples.BasicRawTableOperations <TachyonMasterAddress> <FilePath>");
+      System.out.println("java -cp target/tachyon-" + Version.VERSION
```
Comment (apc999) [inline examples/src/main/java/tachyon/examples/BasicRawTableOperations.java]: seems like a merge regression here

**Comment by apc999** [comment]
one minor comment left.

**Comment by uronce-cc** [comment]
+1 LGTM

File: examples/src/main/java/tachyon/examples/BasicRawTableOperations.java (hunk 1 above)
Comment (calvinjia) [inline examples/src/main/java/tachyon/examples/BasicRawTableOperations.java]: Good catch, thanks.

**Comment by calvinjia** [comment]
@apc999 Resolved the conflict, thanks.

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: examples/src/main/java/tachyon/examples/BasicRawTableOperations.java [hunk 2]
```diff
@@ -49,10 +50,10 @@
   private long mId;
 
   public BasicRawTableOperations(TachyonURI masterAddress, TachyonURI tablePath,
-      WriteType writeType) {
+      TachyonStorageType writeType) {
     mMasterAddress = masterAddress;
     mTablePath = tablePath;
-    mWriteType = writeType;
+    mWriteType = writeType.isStore() ? WriteType.MUST_CACHE : WriteType.THROUGH;
```
Comment (gpang) [inline examples/src/main/java/tachyon/examples/BasicRawTableOperations.java]: This looks like it is translating the new types to the old types. Why isn't the new types (and new client) used all along?

Also, shouldn't the test then take both TachyonStorageType and UnderStorageType to accurately represent the old WriteTypes?

**Comment by gpang** [comment]
I just had a question about the types. Thanks!

File: examples/src/main/java/tachyon/examples/BasicRawTableOperations.java (hunk 2 above)
Comment (calvinjia) [inline examples/src/main/java/tachyon/examples/BasicRawTableOperations.java]: Its because the RawTableClient is still using the old read/write types.

Comment (gpang) [inline examples/src/main/java/tachyon/examples/BasicRawTableOperations.java]: Ah, ok. Then, should the outer command line take both TachyonStorageType and UnderStorageType, and all combinations are combined to translate to WriteType?

The reason I am asking is that CACHE_THROUGH is a typical configuration, which would be impossible going forward.

Comment (calvinjia) [inline examples/src/main/java/tachyon/examples/BasicRawTableOperations.java]: Right, I think we should take both under storage and tachyon write type. That will be the next step for refactoring Tachyon examples.

**Comment by gpang** [comment]
Ok, I am fine with adding the UnderStorageType in another PR.

**Comment by gpang** [comment]
LGTM

**Pull request merged**